.gitignore

```diff
@@ -10,4 +10,5 @@ certs/
 ipa-ucdavis-edu.env
 cypress.env.json
 cypress/videos
-cypress/screenshots
\ No newline at end of file
+cypress/screenshots
```

Comment:
These changes to .gitignore shouldn't be in this PR as they aren't strictly related to what it claims to be doing.